draft-ietf-quic-transport.md

```diff
@@ -739,9 +742,10 @@ Source Connection IDs during the handshake.
 
 On first receiving an Initial or Retry packet from the server, the client uses
 the Source Connection ID supplied by the server as the Destination Connection ID
-for subsequent packets.  Once a client has received an Initial packet from the
-server, it MUST discard any packet it receives with a different Source
-Connection ID.
+for subsequent packets.  That means that a client might change the Destination
+Connection ID twice during connection establishment.  Once a client has received
```

Comment:
only twice? Is there text to prevent a server from issuing a retry when a retry token is present in initial?

Reply:
There is nothing stopping the server from sending multiple Retry packets (regardless of which flight they came from, whether they had token, or anything).  However, the client is only going to process one of them.

Reply:
right